src/components/InventoryV2/InventoryListByBatch.tsx

```diff
@@ -86,6 +92,56 @@ export default function InventoryListByBatch({ setReportData }: InventoryListByB
   });
   const [filters, setFilters] = useForm<InventoryFiltersType>({});
 
+  const dispatch = useAppDispatch();
+  const subLocations = useAppSelector(selectSubLocations);
+  const [subLocationNames, setSubLocationNames] = useState<string[]>([]);
+  const [nurseryIds, setNurseryIds] = useState<number[]>([]);
+  const query = useQuery();
+  useEffect(() => {
+    const querySubLocationNames = query
+      .get('subLocationName')
+      ?.trim()
+      ?.split(',')
+      ?.filter((s) => s !== '');
+    setSubLocationNames(querySubLocationNames ?? []);
+    const queryNurseryIds = query
+      .get('facilityId')
+      ?.trim()
+      ?.split(',')
+      ?.filter((n) => n !== '')
+      ?.map((id) => Number(id));
+    setNurseryIds(queryNurseryIds ?? []);
+  }, [query]);
```

Comment:
Thanks for adding this feature @charlie-tf ❤️ 
Added a comment in the Jira, supporting multiple params is better done this [way](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams#duplicate_search_parameters) and using `query.getAll(paramName)`